specification/search/resource-manager/Microsoft.Search/preview/2019-10-01-preview/examples/CreateOrUpdateSharedPrivateLinkResource.json

```diff
@@ -0,0 +1,41 @@
+{
+  "parameters": {
+    "searchServiceName": "mysearchservice",
+    "resourceGroupName": "rg1",
+    "api-version": "2019-10-01-preview",
+    "subscriptionId": "subid",
+    "sharedPrivateLinkResourceName": "testResource",
+    "sharedPrivateLinkResource": {
+      "name": "testResource",
+      "properties": {
+        "requestMessage": "please approve",
+        "groupId": "blob",
+        "privateLinkResourceId": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Storage/storageAccounts/storageAccountName"
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "name": "testResource",
+        "properties": {
+          "requestMessage": "please approve",
```

Comment:
not sure if we internally standardize what message customer sees when we create the PE. In future potentially we can ask customers for what message they want to see on PE Approval request

Reply:
This is actually already part of the input of the request.  I can change the message we have in the example if you want, but customers can put whatever they want here and that's what we'll send off on their behalf.

Reply:
cool. just checked the spec as well.